windows/src/engine/kmtip/kmkey.cpp

```diff
@@ -126,13 +98,13 @@ BOOL CKMTipTextService::_KeymanProcessKeystroke(ITfContext *pContext, WPARAM wPa
     return FALSE;   // I4378
   }
 
-  if (!_CheckKeymanLoaded()) {   // I4274
-    Log(L"_KeymanProcessKeystroke: _CheckKeymanLoaded failed");
+  if (!Keyman32Interface::TIPIsKeymanRunning()) {
```

Comment:
I see that `TIPIsKeymanRunning` is replacing a number of checks that used to be internal code for the `kmtip` module, and that it's an `extern`.  I presume this is provided by `keyman32.dll`, given this part of the description?

> This makes the kmtip.dll build dependent on keyman32.dll; loading kmtip.dll (by selecting a Keyman input method) also loads keyman32.dll.

Reply:
I ask because the definition doesn't appear within this PR's changelog.

Reply:
Yes, see https://github.com/keymanapp/keyman/pull/3772/files#diff-0f6686cdf0b3a08d6ced71b454d40bb11e3d0c7b242536563d056cf16e0b1ffeR63 for the `TIPIsKeymanRunning` definition. A deeper refactor should allow us to eliminate it entirely from kmtip in the future, but I don't want to do that at this point in the release cycle.